### DISCUSSION TITLE
Issue inserting rows into a table

### DISCUSSION THREAD

**Discussion by ghost** [jgraph/drawio#2331] [Q&A] [answered]
Hello!

I've been working on a draw.io document for the past few months that is made up of several tables.  I've had no problems until yesterday, when most of the horizontal lines separating table rows disappeared.  Tables now only render a number of horizontal lines separating the rows equal to the number of vertical lines separating the columns.  I'm getting the same behavior in all of my draw.io documents, including new ones.  I first noticed the issue after exporting the file as a PDF, then when I checked the project, the issue was present there as well.  

For example: If I drop a new table into a blank document, it has 3 columns and 3 rows - there are 2 vertical lines separating the columns inside of the table and 2 horizontal lines separating the rows.  If I click the "Insert Row After" button, it creates space for another row, but it does not add a horizontal line to separate the rows.

![image](https://user-images.githubusercontent.com/92827107/137994609-6dce3c63-1a47-45e8-811f-2321c3b37e2d.png)

If I then click the "Insert Column Right" button, it adds space for another column, draws a vertical line to separate it, and then it also adds a horizontal line separating the row that I made previously.  

![image](https://user-images.githubusercontent.com/92827107/137994977-09d4a4c5-0b3d-4901-93a0-b9201f2266fb.png)

If I delete the column that I just added, then the horizontal line also disappears and it goes back to looking like the first picture.  

This is a problem because most of the tables in my project have more rows than they do columns.  Is this a new bug, or did I somehow turn on a setting that enforces square tables?

**Comment by davidjgraph** [comment]
What version do you see under the help menu?

**Reply by ghost** [reply]
It's version 15.5.6 and I'm using Chrome on Windows 10.  I tested on a different computer and I'm seeing the same behavior.

**Comment by raph-ADF** [comment]
I'm experiencing the same issue. I'm on version 15.5.4 on Chrome/Windows 10.

**Comment by davidjgraph** [comment, marked as answer]
Fixed in 15.5.7

**Reply by ghost** [reply]
Thank you!
